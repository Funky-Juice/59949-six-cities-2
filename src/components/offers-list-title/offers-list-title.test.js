import renderer from 'react-test-renderer';
import OffersListTitle from './offers-list-title';
import offerTestObj from '../../mocks/test-offer';

const mockOffers = [offerTestObj];

it(`OffersListTitle correctly renders`, () => {
  const tree = renderer
    .create(<OffersListTitle
      activeCity={mockOffers[0].city}
      activeOffers={mockOffers}
    />)
    .toJSON();

  expect(tree).toMatchSnapshot();
});
