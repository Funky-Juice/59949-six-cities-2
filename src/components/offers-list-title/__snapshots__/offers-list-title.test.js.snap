// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`OffersListTitle correctly renders 1`] = `
<b
  className="places__found"
>
  1
  Â 
  place
   to stay in 
  Amsterdam
</b>
`;
